app/src/main/java/org/simple/clinic/bp/BloodPressureRepository.kt

```diff
@@ -115,4 +115,18 @@ class BloodPressureRepository @Inject constructor(
   }
 
   fun measurement(uuid: UUID): Single<BloodPressureMeasurement> = Single.fromCallable { dao.getOne(uuid) }
+
+  fun markBloodPressureAsDeleted(bloodPressureMeasurement: BloodPressureMeasurement): Completable {
+    return Completable.fromAction {
+      val deletedBloodPressureMeasurement = bloodPressureMeasurement.copy(
+          deletedAt = Instant.now(clock),
+          syncStatus = SyncStatus.PENDING)
+
+      dao.save(listOf(deletedBloodPressureMeasurement))
+    }
+  }
+
+  fun deletedMeasurementAsStream(bloodPressureMeasurementUuid: UUID): Observable<BloodPressureMeasurement> {
```

Comment:
Most of our repository functions are streams, there is no need to mention it in the function name here.

Let's just use `deletedMeasurements()`?

Reply:
Done.